### PR TITLE
Fix to support PHP > 8.0

### PR DESCRIPTION
**Pull request by 13shivam**
- unit test fix
- opencensus lib fix
- dependabot

### CODE REVIEW

File: ext/opencensus.c.backup [hunk 1]
```diff
@@ -0,0 +1,248 @@
+/*
```
Comment (Sunn-y-Arora) [inline ext/opencensus.c.backup]: what are these backup files ???

File: tests/integration/curl/composer.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
     "require": {
-        "php": "^7.2",
+        "php": ">=7.1",
```
Comment (Sunn-y-Arora) [inline tests/integration/curl/composer.json:3]: guzzle has >= 8, few others have php as >= 7.1, possible to keep it same ?

Comment (13shivam) [reply tests/integration/curl/composer.json:3]: guzzle version went by mistake in composer while testing(we have to write for guzzle 7 as 6 is not supported for php8), that doesn't work, this is ideally is fine.

File: ext/opencensus.c.backup (hunk 1 above)
Comment (13shivam) [reply ext/opencensus.c.backup]: Once Reviewed will delete these files, kept backup incase I messed up ;)

**Pull request merged**